web/src/main/java/com/futurewei/alcor/web/entity/node/NodeInfo.java

```diff
@@ -60,6 +60,9 @@ public class NodeInfo implements Serializable {
     @QuerySqlField(index = true)
     private String ncm_id;
 
+    @JsonProperty("data-path-ip")
+    private String dataPathIp;
+
```

Comment:
@DavidLiu506 Do we need `mac_address` for `data-path-ip`? I think the data path should use mac for L2 forwarding, right?

Reply:
ACA don't need it. Arion use mac_address.

Reply:
@xieus correct me if I am wrong. If ACA doesn't need the mac address, we don't need to figure out how to get mac address from the pod in k8s.

Reply:
That would be save a lot of time for us :).

Reply:
Yes, you are right.

Reply:
@DavidLiu506 Just confirm with @xieus Alcor still require `mac_address` for `data path`. We can reuse the current `mac_address` field for `data path`'s mac address.

Reply:
As discussed, this is an implementation issue, and Dahai will create an issue to track it so that we will follow up later.